cmd/src/main.go

```diff
@@ -20,7 +20,7 @@ Usage:
 
 The options are:
 
-	-config=$HOME/src-config.json    specifies a file containing {"accessToken": "<secret>", "endpoint": "https://sourcegraph.com"}
+	-config=$HOME/src-config.json    specifies a file containing {"accessToken": "<secret>", "endpoint": "https://sourcegraph.com", "runAs": "another-user"}
```

Comment:
I do not think that sudo tokens are really going to be used that often. Additionally, this makes it look like this needs to be specified always even when not using sudo tokens.

Instead, please add a section to the README for how to use sudo tokens.